packages/backend/src/apps/delay/actions/delay-for/index.ts

```diff
@@ -47,8 +49,14 @@ const action: IRawAction = {
     const { delayForUnit, delayForValue } = $.step.parameters
 
     if (isNaN(Number(delayForValue))) {
-      throw new Error(
-        'Invalid delay for value entered, please check that you keyed in a number',
+      const stepErrorName = 'Invalid delay for value entered'
+      const stepErrorSolution =
+        'Click on set up action and check that the value is a valid number.'
+      throw generateStepError(
+        stepErrorName,
+        stepErrorSolution,
+        $.step.position,
+        $.app.name,
```

Comment:
nit: consider inlining `stepErrorName` and `stepErrorSolution` instead of creating the variables, since they are only used once.

Reply:
okay will do so, thanks!